app/src/main/java/com/fstyle/structure_android/MainActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.fstyle.structure_android
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.crashlytics.android.Crashlytics
+import io.fabric.sdk.android.Fabric
+
+/**
+ * Create by DaoLQ
+ */
+class MainActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/9b5e04c04bb1d6f0d27fcd26370775b8e23088fd/app/src/main/java/com/fstyle/structure_android/MainActivity.kt#L13"></span>Unexpected indentation (2) (it should be 4) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>